modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -156,7 +156,6 @@
 			var/mob/living/carbon/human/H = M
 			if(prob(50)) //Less spam
 				to_chat(H, "<span class='love'>Your libido is going haywire!</span>")
```

Comment:
`to_chat(user, [...]`, since we are removing the now useless human var definition.

Reply:
My bad: `to_chat(M, [...]`, user is not around.